firmware/thruna/PDM/src/app/app_powerManager.h

```diff
@@ -1,5 +1,11 @@
 #include <stdbool.h>
 #include "io_efuse.h"
+#include "app_timer.h"
+#include "app_retryHandler.h"
+
+#pragma once
+
+#define FAULT_CURRENT_THRESHOLD 0.5f
```

Comment:
This should also be a config parameter, since it'll be difference for every efuse.